cluster/examples/kubernetes/ceph/common.yaml

```diff
@@ -144,6 +147,17 @@ rules:
       - nodes
       - nodes/proxy
       - services
+      # Rook watches secrets for:
+      # the admission controller (TODO: can we remove this once the admission controller creates the self-signed cert?)
+      # CephClient CRs (TODO: why!? can/should we remove this?)
+      # CephCluster CRs (TODO: why!? can/should we remove this?)
+      # CephFilesystem CRs (TODO: why!? can/should we remove this?)
+      # CephFilesystemMirror CRs (TODO: why!? can/should we remove this?)
+      # CephObjectStore CRs (TODO: why!? can/should we remove this?)
+      # CephObjectStoreUser CRs (TODO: why!? can/should we remove this?)
+      - secrets
+      # Rook watches for changes to the operator configmap
+      - configmaps
```

Comment:
Had to make some changes for the operator's permissions to fix the failing CI. 

The OBC permissions are minimal now, but the operator needs these permissions added to compensate.